cmd/osv-scanner/main.go

```diff
@@ -121,11 +121,11 @@ func run(args []string, stdout, stderr io.Writer) int {
 				Hidden: true,
 			},
 			&cli.BoolFlag{
-				Name:  "experimental-all-packages",
+				Name:  "experimental-show-all-packages",
```

Comment:
Oh I mean the CLI names can be kept as is, just change the struct names. We already do that for some of the other flags.